run-cypress-test.js

```diff
@@ -3,5 +3,13 @@ const server = require('./server/server');
 
 cypress.run()
 .then((results) => {
-  return server.stop()
-});
+  console.log(results.totalFailed);
+  if (results.totalFailed > 0) {
+    return server.fail();
+  }
+  return server.stop();
```

Comment:
Could this then be a one-liner?

`results.totalFailed > 0 ? server.fail() : server.stop()`

That would get rid of the log though.